assets/js/components/AdminMenuTooltip/useShowTooltip.js

```diff
@@ -60,15 +60,16 @@ export function useShowTooltip( tooltipStateKey ) {
 			};
 		}
 
-		setValue( tooltipStateKey, {
+		setValue( 'admin-menu-tooltip', {
```

Comment:
Using this hook in multiple places will cause these values to get overwritten, possibly in really weird ways, won't it? Since the hook just sets the value of this fixed key… couldn't this even lead to an infinite re-render since setting this value might cause a component to re-render and eventually trigger calling this hook again?

Not sure about the last part, but seems like this hook definitely isn't supposed to be called from two separate places in the app. That seems like a bad design, especially without any warnings/errors/etc. about being invoked from two places/checks around it.

Reply:
As @aaemnnosttv mentioned on the call yesterday off hand, sharing the key in the store prevent displaying multiple tooltips. ie. if you dismiss multiple notices with the previous tooltip open the new tooltip will simply replace the previous one in the store and on the screen.

I don't see this infinite render condition, the later call to `showTooltip` will simply "win out" and that notifications tooltip will be displayed. `showTooltip` is called on user interactions never on component mount etc. Can you think of a specific case where this would be an issue?

Reply:
Nope, just checking to make sure that wouldn't be the case.

It's probably better to prevent subsequent `useShowTooltip` invocations to the same "slug"/ID from overriding things and instead just ignore them, so there isn't the potential for flickering/weird behaviour. What do you think?

Reply:
I've filed #10575 to explore this further.